packages/menu/src/SubMenu/SubMenu.spec.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  render,
+  waitFor,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { Menu, MenuItem, SubMenu } from '..';
+
+const SubMenuTestWrapper = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <Menu open={true} setOpen={jest.fn()}>
+      {children}
+      <SubMenu title="Second SubMenu" data-testid="sub-menu-b">
+        <MenuItem data-testid="sub-menu-item-b">SubMenu Item Two</MenuItem>
+      </SubMenu>
+    </Menu>
+  );
+};
+
+describe('packages/menu/sub-menu', () => {
+  test('renders a SubMenu open by default, when the SubMenu is active', () => {
+    const { getByTestId } = render(
+      <SubMenuTestWrapper>
+        <SubMenu data-testid="sub-menu-a" active={true} />
+      </SubMenuTestWrapper>,
+    );
+
+    // TODO: The submenu is closed on initial render, but opens on second render
```

Comment:
what is the todo?

Reply:
Updated the comment. There's redundant rendering in Menu that needs to be fixed.

Opened a ticket https://jira.mongodb.org/browse/LG-2904